src/pocket/PocketVPC.spec.ts

```diff
@@ -1,11 +1,13 @@
-import { Testing, TerraformStack } from 'cdktf';
+import { Testing } from 'cdktf';
 import { PocketVPC } from './PocketVPC';
+import { DataAwsVpc } from '@cdktf/provider-aws';
+import 'cdktf/lib/testing/adapters/jest';
 
 test('renders a VPC with minimal config', () => {
-  const app = Testing.app();
-  const stack = new TerraformStack(app, 'test');
+  const synthed = Testing.synthScope((stack) => {
+    new PocketVPC(stack, 'testPocketVPC');
+  });
 
-  new PocketVPC(stack, 'testPocketVPC');
-
-  expect(Testing.synth(stack)).toMatchSnapshot();
+  expect(synthed).toMatchSnapshot();
+  expect(synthed).toHaveDataSource(DataAwsVpc);
```

Comment:
This is pretty cool. @bkochendorfer I think you'll find this interesting as well

Reply:
This is cool and it looks like we get a couple other helpful testing matchers too!

```
    toHaveResource: Checks if a certain resource exists
    toHaveResourceWithProperties: Checks if a certain resource exists with all properties passed
    toHaveDataSource: Checks if a certain data source exists
    toHaveDataSourceWithProperties: Checks if a certain data source exists with all properties passed
```